rpcs3/Crypto/unself.cpp

```diff
@@ -1293,10 +1293,10 @@ bool SELFDecrypter::GetKeyFromRap(u8* content_id, u8* npdrm_key)
 
 	// Try to find a matching RAP file under exdata folder.
 	const std::string ci_str = reinterpret_cast<const char*>(content_id);
-	const std::string rap_path = "/dev_hdd0/home/" + Emu.GetUsr() + "/exdata/" + ci_str + ".rap";
+	const std::string rap_path = Emulator::GetHddDir() + "/home/" + Emu.GetUsr() + "/exdata/" + ci_str + ".rap";
```

Comment:
Would be good if you could add also checking for .edat files since for now users are renaming some licenses to .rap to make them work.

Reply:
https://github.com/RPCS3/rpcs3/issues/7066
just wrote sth to that, that doesnt work like that, the games are even unlocked with a stub as long the name is correct, the issue lies sth else, so pls dont spread wrong info.